BAZProject/MainMovie/MainMoviePresenter.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  MainMoviePresenter.swift
+//  BAZProject
+//
+//  Created 1030364 on 18/10/22.
+//  Copyright © 2022 ___ORGANIZATIONNAME___. All rights reserved.
+//
+//  Template generated by Juanpe Catalán @JuanpeCMiOS
+//
+
+import UIKit
+
+class MainMoviePresenter: MainMoviePresenterProtocol {
+
+    weak private var view: MainMovieViewProtocol?
+    var interactor: MainMovieInteractorProtocol?
```

Comment:
Si estás inyectando esta propiedad por inicializador, no es necesario que sea opcional ni que sea internal. Considera cambiarla de la siguiente forma:
```suggestion
    private let interactor: MainMovieInteractorProtocol
```